c-dependencies/js-compute-runtime/js-compute-builtins.cpp

```diff
@@ -1423,6 +1423,8 @@ bool set_cache_key(JSContext *cx, HandleObject self, HandleValue cache_key_val)
   std::string_view key(keyString.get(), key_len);
   std::string hex_str;
   picosha2::hash256_hex_string(key, hex_str);
+  std::transform(hex_str.begin(), hex_str.end(), hex_str.begin(),
+                 [](unsigned char c) { return std::toupper(c); });
```

Comment:
Nice. I didn't realize it was acceptable to use the input range beginning as the output range beginning, but the docs call that out explicitly: https://en.cppreference.com/w/cpp/algorithm/transform.